coreos-base/hard-host-depends/hard-host-depends-0.0.1.ebuild

```diff
@@ -163,6 +163,11 @@ RDEPEND="${RDEPEND}
 	virtual/cdrtools
 	"
 
+# Host dependencies for pre-populating rkt container images
+RDEPEND="${RDEPEND}
+	app-emulation/rkt
```

Comment:
Shall we disable all stage1s for the host dependency explicitly? This would shrink the build environment, if that is something we care about.